spf.gemspec

```diff
@@ -38,35 +39,31 @@ Gem::Specification.new do |s|
   ]
   s.homepage = "https://github.com/agaridata/spf-ruby"
   s.licenses = ["none (all rights reserved)"]
-  s.require_paths = ["lib"]
-  s.rubygems_version = "1.8.23.2"
+  s.rubygems_version = "2.4.6"
```

Comment:
Is it a requirement? I think it just notes the version used generate the gemspec. It looks like this set s the requirement `s.required_rubygems_version = Gem::Requirement.new(">= 0")`